zeal/main.go

```diff
@@ -39,7 +46,22 @@ for the Paladin skill in Diablo II since its job is to bang things down as fast
 }
 
 func main() {
-	_, info := flags.ParseFlagsOrDie("Zeal", &opts, &opts.Logging)
-	go flags.ServeAdmin(opts.Admin, info)
-	rpc.ServeForever(opts.GRPC, opts.Storage.Storage, opts.Storage.TLS, opts.Parallelism)
+	_, info := cli.ParseFlagsOrDie("Zeal", &opts, &opts.Logging)
+	go cli.ServeAdmin(opts.Admin, info)
+
+	for domain, headers := range opts.Headers {
+		for name, header := range headers {
+			log.Notice("Applying header for %s: %s: %s", domain, name, header)
```

Comment:
Is there meant to be a function call or something to go with this log line?

Reply:
Nope - just noting what the config is that we loaded (since the flag structure is fairly complex it seems useful to have a record of it at startup)

Reply:
I've changed the log message so it's a little more appropriate about what it's doing (or not doing) at this point.